src/components/FormattedBytes/utils.tsx

```diff
@@ -11,5 +11,5 @@ export const toFormattedSize = (
         return null;
     }
 
-    return <FormattedBytes value={value} significantDigits={2} {...params} />;
+    return <FormattedBytes value={value} {...params} />;
```

Comment:
It seems we don't use `significantDigits` param any more. What do you think about remove it completely? It seems it will simplify logic for formatting values a lot.